int8 ACTION_READ=1
int8 ACTION_WRITE=2
int8 relay_id
int8 action
int8 write_state
---
int8 relay_state
bool success
